mk-tar.sh

```diff
@@ -95,6 +95,12 @@ function build_curvefs_python() {
 
         bash ./curvefs_python/configure.sh $(basename ${bin})
 
+        # Backup tmp so files
```

Comment:
Maybe it's better to copy so files to `./build` dir directory. And, `mk-deb.sh` also has same problem。